src/libraries/System.Private.CoreLib/src/System/Reflection/StrongNameKeyPair.cs

```diff
@@ -29,9 +29,11 @@ public StrongNameKeyPair(byte[] keyPairArray)
         protected StrongNameKeyPair(SerializationInfo info, StreamingContext context) =>
```

Comment:
This and other serialization related APIs are exposed in the ref maybe we need to annotate them too CC @GrabYourPitchforks

Reply:
Yeah why was that not considered here? Is it because the exception doesn't have the StrongNameSigning message?

Reply:
Now i think we should obsolete this StrongNameKeyPair entirely, all its members just throwing, except one constructor `public StrongNameKeyPair(FileStream keyPairFile)` which also doesn't seem like doing anything cc @steveharter

Reply:
It is only used here https://github.com/dotnet/runtime/blob/32a86674e4942aeb76cba4d6a689aa207ae237fe/src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs#L203-L207 which doesn't seem used anywhere in runtime

Reply:
Buyaa pointed this out to me offline as well. I agree with her assessment that the entire type can go away. I'd also support removing the code in the ctor and replacing it with a "throw" statement like the other methods do.